include/graphblas/reference/blas3.hpp

```diff
@@ -1204,6 +1204,210 @@ namespace grb {
 			return SUCCESS;
 		}
 
+		template<
+			bool upper,
+			Descriptor descr = descriptors::no_operation,
+			typename InputType, typename OutputType,
+			typename RIT_L, typename CIT_L, typename NIT_L,
+			typename RIT_A, typename CIT_A, typename NIT_A
+		>
+		RC trilu_generic(
+			Matrix< OutputType, reference, RIT_L, CIT_L, NIT_L > &L,
+			const Matrix< InputType, reference, RIT_A, CIT_A, NIT_A > &A,
+			const long int k,
+			const Phase &phase
```

Comment:
if taking a lambda-function approach to resolving local vs. global indexing, this is probably where it should go. I'll hold off a deep review on this function until it's clear whether or not such a thing will be added=)